package.json

```diff
@@ -221,17 +221,18 @@
     "@babel/preset-typescript": "^7.18.6",
     "@babel/register": "^7.4.4",
     "@fortawesome/fontawesome-free": "^5.8.1",
-    "@nrwl/cli": "^15.0.0",
-    "@nrwl/eslint-plugin-nx": "^15.0.0",
-    "@nrwl/jest": "15.0.0",
-    "@nrwl/js": "15.0.0",
-    "@nrwl/linter": "15.0.0",
-    "@nrwl/node": "15.0.0",
-    "@nrwl/react": "15.0.0",
-    "@nrwl/tao": "^15.0.0",
-    "@nrwl/web": "15.0.0",
-    "@nrwl/webpack": "15.0.0",
-    "@nrwl/workspace": "^15.0.0",
+    "@nrwl/cli": "^15.6.3",
+    "@nrwl/eslint-plugin-nx": "^15.6.3",
+    "@nrwl/jest": "15.6.3",
+    "@nrwl/js": "15.6.3",
+    "@nrwl/linter": "15.6.3",
+    "@nrwl/node": "15.6.3",
+    "@nrwl/react": "15.6.3",
+    "@nrwl/tao": "^15.6.3",
+    "@nrwl/web": "15.6.3",
+    "@nrwl/webpack": "15.6.3",
+    "@nrwl/workspace": "^15.6.3",
+    "@svgr/webpack": "^6.5.1",
```

Comment:
is this dep required?

Reply:
you mean @svgr/webpack?
yes...

Reply:
at least I got errors without it. can check again.